src/main.rs

```diff
@@ -280,8 +280,10 @@ fn main() {
         create_config_and_exit(enable_styles);
     }
 
-    // Specify target OS
-    let platform: PlatformType = args.platform.unwrap_or_else(PlatformType::current);
+    let platforms = match args.platform {
+        Some(ref platforms) => platforms.clone(),
+        None => vec![PlatformType::current()],
+    };
```

Comment:
We can avoid the clone here by borrowing into the `Option`:
```suggestion
    const FALLBACK_PLATFORMS: &'static [PlatformType] = &[PlatformType::current()];
    let platforms = args
        .platform
        .as_ref()
        .map_or(FALLBACK_PLATFORMS, Vec::as_slice);
```